.mocharc.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
do not delete anything which is merged to main already

Reply:
sure.